package-lock.json

```diff
@@ -786,7 +796,7 @@
         },
         "util": {
           "version": "0.10.3",
-          "resolved": "https://registry.npmjs.org/util/-/util-0.10.3.tgz",
+          "resolved": "http://registry.npmjs.org/util/-/util-0.10.3.tgz",
```

Comment:
Do not change to http

Reply:
I did not do that, npm did that.
As it doe not have impact on end user. we could let it go.